blocks/shared-styles/_children/headings/index.test.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import Heading from './heading';
+import HeadingSection from './section';
+
+jest.mock('fusion:context', () => ({
+  useFusionContext: jest.fn(() => ({
+    globalContent: {},
+    arcSite: 'the-sun',
+  })),
+}));
+
+jest.mock('fusion:themes', () => jest.fn(() => ({})));
+
+describe('Heading', () => {
+  it('should render as a h2', () => {
+    const wrapper = mount(<Heading />);
+
+    expect(wrapper.html()).toMatchInlineSnapshot(
+      '"<h1 class=\\"sc-bdVaJa bFuGRU\\"></h1>"',
```

Comment:
I forget what we talked about using styled-generated classnames. I guess it'll update with snapshots anyway if we change it

Reply:
This should only change if the PrimaryFont component changes - which shouldn't be often